### ISSUE TITLE
Security Event GUI: Cannot add many "Endpoint" and "Device profiling" trigger parts

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#4348]
In the security event trigger line, we should be able to select multiple values in the "Endpoint" and the "Device profiling" columns for a row.

For example, I should be able to make my trigger fire on Endpoint->Role->default and Endpoint->Switch->192.168.1.1

**Comment by cgx** [comment]
If I have this trigger defined:

```json
[1100003]
trigger=mac_vendor::42,(device::2808&device::86)
...
```

The API give me this JSON representation (**missing device 2808**):

```bash
$ /usr/local/pf/sbin/pfperl-api get /api/v1/config/security_event/1100003 | jq '.item.triggers|delpaths([path(.[][]| select(.==null))])'
[
  {
    "device": [],
    "mac_vendor": "42"
  },
  {
    "device": [
      "86"
    ]
  }
]
```

Can you @julsemaan or @jrouzierinverse fix the endpoint to return the correct values?

**Comment by julsemaan** [comment]
There can only be one device sub-trigger so device::2808 and device::86 isn't valid

For device profiling (and endpoint), there can be multiple different sub triggers but there cannot be multiple of them since it wouldn't make sense to look if the DHCP fingerprint is equal to something and something else which will never be a true value

**Comment by cgx** [comment]
All right. I'll adjust the GUI.

**Comment by cgx** [comment]
Fun fact: I've built the invalid trigger condition with the old Web admin.

**Comment by julsemaan** [comment]
Man that guy who wrote the triggers tab in the old admin, don't start me on him :)

**Comment by cgx** [comment]
Partially fixed via 7b1c469.
